### ISSUE TITLE
为什么bert原本的只有MLP层，accuracy能达到86%，加了BiLSTM+Attention却降到了65%

### ISSUE THREAD

**Issue by SmartMapple** [jiangxinyang227/textClassifier#21]
加BiLSTM+Attention
INFO:tensorflow:***** Eval results *****
INFO:tensorflow:  **eval_accuracy = 0.6544118**
INFO:tensorflow:  eval_auc = 0.52433944
INFO:tensorflow:  eval_precision = 0.69602275
INFO:tensorflow:  eval_recall = 0.8781362
INFO:tensorflow:  global_step = 917
INFO:tensorflow:  loss = 0.77603155

bert原生的MLP层
![官方参数 but 0 5 dropout keep](https://user-images.githubusercontent.com/44792001/102893168-11412100-449c-11eb-8127-47c214db4929.PNG)

**Comment by SmartMapple** [comment]
it's because of my code. now it works well.
